Remotion/Security/UnitTests/Metadata/LocalizationFileNameStrategyTest.cs

```diff
@@ -96,8 +96,8 @@ public void GetLocalizationFileNames_WithoutBaseDirectory ()
 
       Assert.That(localizationFileNames, Is.Not.Null);
       Assert.That(localizationFileNames.Length, Is.EqualTo(2));
-      Assert.That(localizationFileNames, Has.Member(@".\TwoLocalizationFilesIncludingInvariantCulture.Localization.de.xml"));
-      Assert.That(localizationFileNames, Has.Member(@".\TwoLocalizationFilesIncludingInvariantCulture.Localization.xml"));
+      Assert.That(localizationFileNames, Has.Member(Path.Combine(".", "TwoLocalizationFilesIncludingInvariantCulture.Localization.de.xml")));
```

Comment:
is Path.Combine doing the same thing ehre as the original hard-coded string or is it trimming the path?

Reply:
What do you mean with "trimming the path"? `Path.Combine` just does $"{left}{Path.DirectorySeparatorChar}{right}" but optimized.

Reply:
`.\file.xml` and `file.xml` are equivalent. The question is, whe you do Path.Combine with a `.`, does dotnet actually prefix with the dot-folder or does it only substitute the file name. Since we're testing that the `,\` is in there, using manual concatenation with the PathSepChar might be more explicit / less prone to question if the test tests the correct thing?